src/main/webapp/resources/js/components/samples/components/SampleAnalyses.jsx

```diff
@@ -1,43 +1,153 @@
 import React from "react";
-import { Input, Space, Table } from "antd";
+import { Col, Input, Row, Table, Typography } from "antd";
+import { useDispatch, useSelector } from "react-redux";
+import { fetchSampleAnalyses } from "../../../apis/samples/samples";
+import { setSampleAnalyses } from "../sampleAnalysesSlice";
+
+import { setBaseUrl } from "../../../utilities/url-utilities";
+import { formatInternationalizedDateTime } from "../../../utilities/date-utilities";
+import {
+  IconCheckCircle,
+  IconClock,
+  IconCloseCircle,
+  IconLoading,
+} from "../../icons/Icons";
+import { blue6, green6, grey6, red6 } from "../../../styles/colors";
 
 const { Search } = Input;
+const { Paragraph } = Typography;
+const commonIconStyle = {
+  fontSize: "16px",
+};
+
 /**
  * React component to display sample analyses
  *
  * @returns {JSX.Element}
  * @constructor
  */
 export function SampleAnalyses() {
+  const [filteredSubmissions, setFilteredSubmissions] = React.useState(null);
+  const dispatch = useDispatch();
+
+  const { sample } = useSelector((state) => state.sampleReducer);
+
+  const { analyses, loading } = useSelector(
+    (state) => state.sampleAnalysesReducer
+  );
+
+  /*
+  On page load get the sample analyses
+   */
+  React.useEffect(() => {
+    fetchSampleAnalyses({ sampleId: sample.identifier })
+      .then((analysesList) => {
+        dispatch(setSampleAnalyses({ analyses: analysesList }));
+      })
+      .catch((error) => {});
```

Comment:
Are you doing anything with this error?

Reply:
Was not being used for anything. Removed in 3cc32f03e78bb36dfc7aba5f0f1fa4d2fe94c0a6